tests/esp32/wait_for2.toit

```diff
@@ -17,10 +17,12 @@ main:
   pin_in := gpio.Pin PIN_IN --input
   pin_out := gpio.Pin PIN_OUT --output
 
-  ITERATIONS.repeat:
+  ITERATIONS.repeat: | iteration |
+    if iteration % 1000 == 0: print "Iteration: $iteration"
```

Comment:
Is the printing still useful?

Reply:
I find it useful, as it shows me if that tests are doing something and not just hitting the timeout.